src/kakarot/accounts/library.cairo

```diff
@@ -99,25 +102,28 @@ namespace AccountContract {
         return ();
     }
 
-    // @notice This function is used to upgrade the smart contract account.
+    // @notice Upgrades the implementation of the account.
```

Comment:
I think that trunk doesn't catch this on cairo file but it should be "Upgrade" (try it on a python docstring)